### DISCUSSION TITLE
Pympress on Surface tablet

### DISCUSSION THREAD

**Discussion by ferdinand-fichtner** [Cimbali/pympress#215] [Q&A]
Hi all,

I hope it is acceptable to use this board for exchange between Pympress users; if not, please let me know, admins!

Does anyone have experience with using Pympress on a MS Surface Tablet with active pen? I am currently using pympress on my (MS Windows) laptop and am extremely happy, except when drawing in my slides. While my Laptop has a touch screen, the precision when annotating the slides is far from acceptable. 

I am now considering investing into a Surface Pro other Tablet PC which allows the use of an active pen. Can anyone give recommendations as to how well Pympress works on these tablet PCs and regarding precision when annotating slides?

Thank you
Ferdinand

**Comment by Cimbali** [comment]
Hi @ferdinand-fichtner I’ve moved this to discussions which is a new github feature, and seems a better place than issues for your question.

Though I’m afraid nobody answered this for 18 months so maybe recommendations would be outdated for your investment. ;)
Did you end up trying this yourself?
